src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -16,9 +16,13 @@
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
+import frc.robot.vision.PhotonVision;
+
 import static frc.robot.Constants.SwerveConstants.*;
 
 public class SwerveSubsystem extends SubsystemBase {
+    private final PhotonVision vision;
```

Comment:
small detail, but perhaps this class should be called "PhotonVisionConnection" or something for clarity?